Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/PDController.java

```diff
@@ -343,14 +304,17 @@ else if (mainAttribute.getValueSize() > configData.getMaxNumberOfUniqueValues())
             graphSettingsController = pdFactory.createGraphSettingsController(this);
             logStatsController = pdFactory.createLogStatsController(this);
             timeStatsController = pdFactory.createTimeStatsController(this);
+            logFilterController = pdFactory.createLogFilterController(this);
+            animationController = pdFactory.createAnimationController(this);
+            logExportController = pdFactory.createLogExportController(this);
+            bpmnExportController = pdFactory.createBPMNExportController(this);
+            toolbarController = pdFactory.createToolbarController(this);
 
             initialize();
-            initializeDefaults();
-
-            LOGGER.debug("Session ID = " + ((HttpSession)Sessions.getCurrent().getNativeSession()).getId());
-            LOGGER.debug("Desktop ID = " + getDesktop().getId());
-
-            // Finally, store objects to be cleaned up when the session timeouts
+            System.out.println("Session ID = " + ((HttpSession)Sessions.getCurrent().getNativeSession()).getId());
+            System.out.println("Desktop ID = " + getDesktop().getId());
```

Comment:
Convert these back to `LOGGER.debug` if they're to be kept.

Reply:
Ok